src/assets.ts

```diff
@@ -52,22 +50,284 @@ interface AttachConfiguration extends DebugConfiguration {
+        if (!isWebProject) {
+            return {
+                version: version,
+                configurations: [
+                    this.createLaunchConfiguration(),
+                    this.createAttachConfiguration()
+                ]
+            };
+        }
+        else {
+            return {
+                version: version,
+                configurations: [
+                    this.createWebLaunchConfiguration(),
+                    this.createAttachConfiguration()
+                ]
+            };
+        }
+    }
+
+    private createBuildTaskDescription(): tasks.TaskDescription {
+        let buildPath = '';
+        if (this.hasProject) {
+            buildPath = path.join('${workspaceRoot}', path.relative(this.rootPath, this.projectFilePath));
+        }
+
+        return {
+            taskName: 'build',
+            args: [buildPath],
+            isBuildCommand: true,
+            problemMatcher: '$msCompile'
+        };
+    }
+
+    public createTasksConfiguration(): tasks.TaskConfiguration {
+        return {
+            version: '0.1.0',
+            command: 'dotnet',
+            isShellCommand: true,
+            args: [],
+            tasks: [this.createBuildTaskDescription()]
+        };
+    }
+}
+
+function findNetCoreAppTargetFramework(project: protocol.MSBuildProject) {
+    return project.TargetFrameworks.find(tf => tf.ShortName.startsWith('netcoreapp'));
 }
 
-function getPaths(): Paths {
-    const vscodeFolder = path.join(vscode.workspace.rootPath, '.vscode');
+function findExecutableMSBuildProjects(projects: protocol.MSBuildProject[]) {
+    let result: protocol.MSBuildProject[] = [];
 
-    return {
-        vscodeFolder: vscodeFolder,
-        tasksJsonPath: path.join(vscodeFolder, 'tasks.json'),
-        launchJsonPath: path.join(vscodeFolder, 'launch.json')
-    };
+    projects.forEach(project => {
+        if (project.IsExe && findNetCoreAppTargetFramework(project) !== undefined) {
+            result.push(project);
+        }
+    });
+
+    return result;
 }
 
+function findExecutableProjectJsonProjects(projects: protocol.DotNetProject[], configurationName: string) {
+    let result: protocol.DotNetProject[] = [];
+
+    projects.forEach(project => {
+        project.Configurations.forEach(configuration => {
+            if (configuration.Name === configurationName && configuration.EmitEntryPoint === true) {
+                if (project.Frameworks.length > 0) {
+                    result.push(project);
+                }
+            }
+        });
+    });
+
+    return result;
+}
+
+function containsDotNetCoreProjects(workspaceInfo: protocol.WorkspaceInformationResponse) {
+    if (workspaceInfo.DotNet && findExecutableProjectJsonProjects(workspaceInfo.DotNet.Projects, 'Debug').length > 0) {
+        return true;
+    }
+
+    if (workspaceInfo.MsBuild && findExecutableMSBuildProjects(workspaceInfo.MsBuild.Projects).length > 0) {
+        return true;
+    }
+}
```

Comment:
What? It's JavaScript. Of course not!

And, yes, I clearly missed that. Thanks!